src/main/kotlin/build/buf/gradle/ProtobufGradlePluginSupport.kt

```diff
@@ -79,10 +80,22 @@ abstract class WriteWorkspaceYamlTask : DefaultTask() {
 
     @TaskAction
     fun writeWorkspaceYaml() {
-        val protoDirs = allProtoDirs().map { project.makeMangledRelativizedPathStr(it) }
-        val bufYaml = bufYamlGenerator.generate(project.bufConfigFile(), protoDirs)
-        logger.info("Writing generated buf.yaml:{}\n", bufYaml)
-        File(bufbuildDir, "buf.yaml").writeText(bufYaml)
+        if (Version(project.getExtension().toolVersion) < Version("1.32.0")) {
```

Comment:
Looks like we care about `Version(project.getExtension().toolVersion) < Version("1.32.0")` in more than one place. Could be worth extracting to a constant or helper method of some kind.

Reply:
Done